packages/@react-types/shared/src/events.d.ts

```diff
@@ -12,29 +12,27 @@
 
 import {
   FocusEvent,
-  KeyboardEvent as ReactKeyboardEvent,
+  KeyboardEvent,
   SyntheticEvent
 } from 'react';
 
 // Event bubbling can be problematic in real-world applications, so the default for React Spectrum components
 // is not to propagate. This can be overridden by calling continuePropagation() on the event.
-export type BaseEvent<T extends SyntheticEvent> = T & {
+export type BaseEvent<T extends SyntheticEvent<HTMLElement>> = T & {
   /** @deprecated Use continuePropagation. */
   stopPropagation(): void,
   continuePropagation(): void
 }
 
-export type KeyboardEvent = BaseEvent<ReactKeyboardEvent<any>>;
```

Comment:
Not sure why this existed, but removing it fixed some issues.

Reply:
I don't think we can remove this export, it's in a published package.
I imagine it's also the reason for your docs build failing https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/interactions/src/usePress.ts#L273